examples/echobot.py

```diff
@@ -25,29 +28,29 @@ def main():
 
     while True:
         try:
-            update_id = echo(bot, update_id)
+            echo(bot)
         except NetworkError:
             sleep(1)
         except Unauthorized:
             # The user has removed or blocked the bot.
             update_id += 1
 
 
-def echo(bot, update_id):
-
+def echo(bot):
+    global update_id
     # Request updates after the last update_id
     for update in bot.getUpdates(offset=update_id, timeout=10):
         # chat_id is required to reply to any message
         chat_id = update.message.chat_id
         update_id = update.update_id + 1
+        if not update.message:  # we ignore updates without messages
+            continue
         message = update.message.text
 
         if message:
```

Comment:
Maybe for simplicity we could go with (and remove the ``if not update.message``):
```
if update.message:
   bot.sendMessage(chat_id=chat_id, text=update.message.text)
```

But it's your call.